src/main/java/com/google/sps/utility/GeocodingResultUtility.java

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.utility;
+
+import com.google.maps.model.AddressType;
+import com.google.maps.model.GeocodingResult;
+import com.google.maps.model.LatLng;
+import com.google.maps.model.PlaceType;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+/** Utility class to extract data from GeocodingResult objects. */
+public class GeocodingResultUtility {
+  /**
+   * Parses for the first coordinate which is of a street address type in a result from the
+   * Geocoding API. If no street address type results are found, the coordinates of the first result
+   * is returned.
+   *
+   * @param result A GeocodingResult returned from the Geocoding API.
```

Comment:
Javadoc does not match the header here - change parameter name

Reply:
Done, thanks!

Reply:
`RESOLVED`